src/commands/metrics.ts

```
@@ -0,0 +1,360 @@
+        // Load and analyze functions from index
+        const indexContent = fs.readFileSync(config.output, 'utf-8');
+        const functions: any[] = [];
+        
+        for (const line of indexContent.trim().split('\n').filter(line => line)) {
+          try {
+            functions.push(JSON.parse(line));
+          } catch (error) {
+            console.warn(chalk.yellow(`⚠️  Skipping malformed JSON line: ${line.substring(0, 50)}...`));
+            continue;
+          }
+        }
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)